config/deploy/production.rb

```diff
@@ -1,4 +1,14 @@
 role :web, %w{deploy@beta.prx.org}
 set :deploy_to, '/var/www/domains/prx.org/m'
 set :branch, :master
-set :default_env, { path: "/opt/node/current/bin:/opt/python/current/bin:$PATH" }
+set :default_env, { path: '/opt/node/current/bin:/opt/python/current/bin:$PATH' }
+
+set :slack_webhook, -> {
+  webhook = nil
+  on roles(:web) do
+    cmd = 'cat /var/www/domains/prx.org/hal/shared/config/slack_webhook.txt'
```

Comment:
This should probably be pulling from the deploy location for this app rather than for another app.